modules/tinymce/src/plugins/paste/main/ts/api/Settings.ts

```diff
@@ -45,6 +45,20 @@ const shouldConvertWordFakeLists = (editor: Editor): boolean => editor.getParam(
 
 const shouldUseDefaultFilters = (editor: Editor): boolean => editor.getParam('paste_enable_default_filters', true);
 
+const getValidate = (editor: Editor) => editor.getParam('validate');
+
+const getAllowHtmlDataUrls = (editor: Editor): boolean => editor.getParam('allow_html_data_urls', false, 'boolean');
+
+const getPasteDataImages = (editor: Editor): boolean => editor.getParam('paste_data_images', false, 'boolean');
+
+const getImagesDataimgFilter = (editor: Editor) => editor.getParam('images_dataimg_filter');
```

Comment:
Intentional. Didn't put it in capital since it was "images_dataimg_filter" rather than "images_data_img_filter", and I thought it might have some reason for not being so. Might been a mistake on my part though?

Reply:
I'm going to blame that on the setting being badly named 😂  We might as well make the variable pretty though, so we're less likely to get it wrong if we're using it in code in the future? So I'd vote `getImagesDataImgFilter`